### PR TITLE
Update registry version to v0.7.1 on production

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Here the version of the registry is specified this storage branch uses.
 # It should always be a specific version to make sure builds are reproducible.
-ARG PACKAGE_REGISTRY=v0.6.0
+ARG PACKAGE_REGISTRY=v0.7.1
```
Comment (ruflin) [inline Dockerfile:3]: @ruflin @jonathan-buttner would be good to know if there was a reason that prod was still on v0.6.0 and not v0.7.0

Comment (jonathan-buttner) [reply Dockerfile:3]: Yeah I'm not sure about that. I see there were PRs to snapshot and staging though 🤔

Comment (ruflin) [reply Dockerfile:3]: Yes, staging and snapshot were already on 0.7.0 ...

File: testing/main_integration_test.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ type Package struct {
 }
 
 func getPackages(t *testing.T) ([]string, error) {
-	resp, err := http.Get("http://localhost:8080/search")
+	resp, err := http.Get("http://localhost:8080/search?experimental=true")
```
Comment (ruflin) [inline testing/main_integration_test.go:131]: @ph @jonathan-buttner Looking at the diff between staging / production, this is what I found.

File: Dockerfile (hunk 1 above)
Comment (ph) [reply Dockerfile:3]: The only reason what  not to include a breaking change in production.

Comment (ruflin) [reply Dockerfile:3]: @ph What is this breaking change?

**Comment by ruflin** [comment]
I finally got this working. The problem was that the packages in production are not up-to-date yet and will not be installed on master. But this unconvered an other problem. The tests are currently run against master. This is one option. But I think instead we should run it at the current or next minor as this what it will be shipped with. I'll open a separate issue to discuss on which tests should be run.

File: testing/main_integration_test.go [hunk 3]
```diff
@@ -128,7 +128,7 @@ type Package struct {
 }
 
 func getPackages(t *testing.T) ([]string, error) {
-	resp, err := http.Get("http://localhost:8080/search")
+	resp, err := http.Get("http://localhost:8080/search?experimental=true&kibana.version=8.0.0")
```
Comment (jonathan-buttner) [inline testing/main_integration_test.go:131]: I'm probably misunderstanding some but I thought we were going to be switching the packages and the endpoint package to kibana version `^7.9.0` which would be excluded here right?

So if I need to release a new endpoint package for 7.9 in production that has the kibana version `^7.9.0` would CI here pick it up?

Comment (mtojek) [reply testing/main_integration_test.go:131]: Did anyone consider a switch on the Kibana side? User could select between stable and experimental packages.

Comment (ruflin) [reply testing/main_integration_test.go:131]: @mtojek Yes, that was the initial plan for 7.9. I was now delayed as we enabled experimental packages by default as most of the packages are in experimental.

@jonathan-buttner You are correct with the above and I will open an issue around what version of Kibana we should test against. Currently we test against 8.0.0-SNAPSHOT so I would say the above is correct. Getting this PR in first to have production on a more recent version of the registry and then follow up.

**Comment by ruflin** [comment]
@jonathan-buttner With this being merged, could you chime in on https://github.com/elastic/package-storage/issues/163 Also this change in Kibana should solve your issue: https://github.com/elastic/kibana/pull/73415

**Pull request merged**